postcss.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
As much as possible, I'd like to keep all such configs inside the config directory. Can you try this... move this config to `config/postcss/postcss.config.js` instead, and update webpack with the following:

    loader: ExtractTextPlugin.extract('css-loader?modules&importLoader=1!postcss-loader?config=../postcss/postcss.config.js'),

https://github.com/postcss/postcss-loader/pull/143